docs/toc.md

```diff
@@ -270,7 +270,7 @@
 ## [Tour of C#](csharp/tour-of-csharp/)
 <!-- The "What's New" section is short, and one level
     deep, so leave it in the main TOC -->
-## What's new in C#
+## What's new in C\#
```

Comment:
We have a bug on the TOC files where they follow the escaping rules for Markdown so let's undo this for now. Otherwise, it renders incorrectly:

![image](https://user-images.githubusercontent.com/12971179/53396105-7b4c1600-3958-11e9-9698-13fb1991e5b6.png)

```suggestion
## What's new in C#
```

Reply:
Is that being tracked somewhere public or private?